python/download_gpx_from_sequences.py

```diff
@@ -0,0 +1,213 @@
+            try:
+                
+                image = urllib.URLopener()
+                image.retrieve(url, path+file_captured_at+'_'+filename)
+                write_log(file_captured_at+'_'+filename+','+str(im['captured_at'])+','+str(im['user'])+','+str(im['location']))
+                
+		if START_ESTIMATED_TIME_AFTER >=  counter:
+                        estimated_time = '...'
+                else:
+                        elapsed_time = time.time() - START_TIME
+                        estimated_time = str(timedelta(seconds=(elapsed_time/counter)*(len(query['ss'])-counter))).split(".")[0]
+
+                print("Successfully downloaded: {0} {1}/{2} ET {3}".format(file_captured_at+'_'+filename,counter,len(query['ss']),estimated_time))
+                break
+            except (KeyboardInterrupt, SystemExit):
+                sys.exit()
+            except:
+                retry_counter = retry_counter+1
+                if retry_counter >= MAX_ATTEMPTS:
+                        print("Finally Failed: {0}".format(filename))
+                else:
+                        print("Failed to download (retrying {0}/{1}): {2}".format(retry_counter,MAX_ATTEMPTS,filename))
+            
+    return im_list
+
+def error_exit(extra_txt=''):
+
+	if len(extra_txt)>0:
+		print("Warning:%s" % extra_txt)
+
+	sys.exit("Usage: %s min_lat max_lat min_lon max_lon [max_results] [user_name] [start_time] [end_time]" % sys.argv[0])
+
+
+
+if __name__ == '__main__':
+    '''
+    Use from command line as below
+    '''
+
+    parser = argparse.ArgumentParser(description='Download GPS Traces from Mapillary.',formatter_class=argparse.ArgumentDefaultsHelpFormatter)
+    parser.add_argument('-r','--rect', nargs=4,metavar=('min_lat', 'max_lat', 'min_lon','max_long'),
+                              default=[-90, 90, -180, 180], help='latitude and longitude rectangle')
+    parser.add_argument('-m','--maxresults' , type=int, default=MAX_RESULTS, help='Max results to get from server')
+    parser.add_argument('-u','--username'   , default='', help='Get traces only from user.')
+    parser.add_argument('-d1','--startdate' , type=int, default=DEFAULT_START_TIME, help='Get traces only from this date. YYYYMMDD')
+    parser.add_argument('-d2','--enddate'   , type=int, default=DEFAULT_END_TIME,   help='Get traces until this date only. YYYYMMDD')
+
+    args = parser.parse_args()
+    
+    min_lat = args.rect[0]
+    max_lat = args.rect[1]
+    min_lon = args.rect[2]
+    max_lon = args.rect[3]
+    max_results = args.maxresults
+
+    if max_results == MAX_RESULTS:
+        print("Warning: default max_results=%d" % MAX_RESULTS)
+
+    if args.username != '':
+        user_filter = args.username
+    else:
+        user_filter = USER_FILTER
+        print("Warning: USER NOT SPECIFIED - NOT A GOOD IDEA, trying to list last gpx from all users")
+
+    #validate and set de start_time        
+    limit_start_time = str(args.startdate)
```

Comment:
this validation is only for get the Start Date of the range. I will check the code, no problem.